epub-a11y-meta-guide/1.0/draft/index.html

```diff
@@ -860,11 +860,54 @@ <h4>Organization of values</h4>
 
 					<p><a href="#i18n-terms">Internationalization terms</a> identify those accessibility characteristics
 						of the content which are required for internationalization.</p>
+                    <p><a href="#unsupported-terms">Unsupported terms</a> include those features that are currently not supported in EPUB.</p>
 				</section>
 
 				<section id="structure-navigation-terms">
 					<h4>Structure and navigation terms</h4>
 
+                <section id="metadata-a11y-features-annotations">
+                    <h5>Annotations</h5>
```

Comment:
Weren't we just discussing deprecating this value, though? I thought the argument against it was that the annotations are generally for all readers and rarely address a specific accessibility need.